packages/test/test-end-to-end-tests/src/test/gc/gcContainerRuntimeCompat.spec.ts

```diff
@@ -156,28 +90,11 @@ describeFullCompat.skip("GC summary compatibility tests", (getTestObjectProvider
 
         /**
          * Submits a summary and returns the unreferenced timestamp for all the nodes in the container. If a node is
-         * referenced, the ureferenced timestamp is undefined.
+         * referenced, the unreferenced timestamp is undefined.
          * @returns a map of nodeId to its unreferenced timestamp.
          */
-        async function getUnreferencedTimestamps(
-            summarizerClient: { containerRuntime: ContainerRuntime; summaryCollection: SummaryCollection; },
-            blobHandleExpected?: boolean,
-        ) {
-            const summary = await submitAndAckSummary(provider, summarizerClient, logger);
-            latestAckedSummary = summary.ackedSummary;
-            assert(
-                latestSummaryContext
-                    && latestSummaryContext.referenceSequenceNumber >= summary.summarySequenceNumber,
-                `Did not get expected summary. Expected: ${summary.summarySequenceNumber}. ` +
-                `Actual: ${latestSummaryContext?.referenceSequenceNumber}.`,
-            );
-            assert(latestUploadedSummary !== undefined, "Did not get a summary");
-
-            const gcState = getGCStateFromSummary(latestUploadedSummary, blobHandleExpected);
-            if (blobHandleExpected === true && gcState === undefined) {
-                return true;
-            }
-
+        async function getUnreferencedTimestamps(summaryTree: ISummaryTree) {
+            const gcState = getGCStateFromSummary(summaryTree);
```

Comment:
This is going to break in the future if we ever chunk GC blobs and or submit summary handles and make it default. I don't know if we need to consider this issue in this test now.

Reply:
These tests are currently disabled so I am not worried. When they are enabled,  they should handle these scenarios correctly.